cmd/debugJs.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"encoding/json"
+
+	"os"
+
+	"github.com/StackExchange/dnscontrol/models"
+	"github.com/StackExchange/dnscontrol/pkg/js"
+	"github.com/urfave/cli"
+)
+
+func exit(err error) error {
+	if err == nil {
+		return nil
+	}
+	return cli.NewExitError(err, 1)
+}
+
+var debugJSCommand = &cli.Command{
+	Name:  "debug-js",
+	Usage: "Execute javascript and print generated json",
```

Comment:
s/Execute javascript and print generated json/Output intermediate representation (IR) after JavaScript is executed but before any validation/normalization/g